packages/swingset-theme-hashicorp/src/components/side-nav/index.tsx

```diff
@@ -1,20 +1,46 @@
 import { NavigationTree } from 'swingset/types'
 import Category from './category'
+import { Link } from '../link'
+import { cx } from 'class-variance-authority'
+import { ToggleButton } from './toggle-button'
 
 type SideNavBarProps = {
   categories: NavigationTree
+  isOpen: boolean
+  toggle: () => void
 }
 
-function SideNavigation({ categories }: SideNavBarProps) {
-  
+function SideNavigation({ categories, isOpen, toggle }: SideNavBarProps) {
+  const renderCategories = categories.map((category) => (
+    <Category
+      title={category.title}
+      key={category.title}
+      items={category.children}
+    />
+  ))
 
-  const renderCategories = categories.map(
-    (category) => (
-      <Category title={category.title} key={category.title} items={category.children} />
-    )
+  return (
+    <aside
```

Comment:
I think when this is closed, we want to apply `aria-hidden` and ensure tabbing through the page doesn't highlight links in here.